src/main.c

```diff
@@ -11,16 +11,24 @@
 #include "../include/main.h"
 #include "../include/menu.h"
 #include "../include/randomizer_8_16_64.h"
-
+#include "../include/bubblesort.h"
+#include "../include/insertionsort.h"
 
 void randomizer_debug(){
     
     int **input = (int **)malloc(3 * sizeof(int *));
```

Comment:
Diesen Randomizer DEBUG section würde ich gerne in ein eigenes modul names testing einfügen damit wir weniger im main.c haben. Da wir alle an main. änderungen vornehmen ist die warscheinlichkeit ein merge conflict im main.c zu haben am höchsten. Eine derartige implementierung kommt im nächten pull request: module_testing.
![Screenshot from 2021-12-26 19-58-58](https://user-images.githubusercontent.com/65472083/147418110-8345ff5e-c482-4c7a-8e8e-6a79abd830ed.png)

Das main.c schaut danach so aus :)
Wenn ihr das aber nicht so haben wollt ist diese Änderung nicht notwendig :)

Reply:
nein, passt